ironfish/src/rpc/routes/wallet/create.test.slow.ts

```diff
@@ -88,4 +93,34 @@ describe('Route wallet/create', () => {
 
     expect(scanSpy).toHaveBeenCalled()
   })
+
+  it('should set account createdAt if passed', async () => {
+    const name = uuid()
+    const createdAt = {
+      hash: '00000000000163b1c632498ac8f86cfd33f65c5d4e43e5001c75d5acb4ada4de',
+      sequence: 10,
+    }
+
+    const response = await routeTest.client.wallet.createAccount({
+      name,
+      createdAt,
+    })
+
+    expect(response.status).toBe(200)
+    expect(response.content).toMatchObject({
+      name: name,
+      publicAddress: expect.any(String),
+      isDefaultAccount: true,
+    })
+
+    const account = routeTest.node.wallet.getAccountByName(name)
+    expect(account).toMatchObject({
+      name: name,
+      publicAddress: response.content.publicAddress,
+      createdAt: {
+        hash: Buffer.from(createdAt.hash, 'hex'),
+        sequence: createdAt.sequence,
+      },
+    })
+  })
 })
```

Comment:
This file was for testing /create but someone changed it to use the new RPC route which means the old /create isn't being tested. On top of that, the file name is now wrong.

Reply:
Ok we're going to fix this in another PR.